src/function.ts

```diff
@@ -13,3 +13,11 @@ export function batchInvoke(functions: Nullable<Fn>[]) {
 export function invoke(fn: Fn) {
   return fn()
 }
+
+/**
+ * Pass the value through the callback, and return the former.
```

Comment:
Let's have a simple example here in the jsdoc, or would feel `tap` could be a bit confusing when out of context.